syntax = "proto3";

option go_package = "github.com/mkatch/spejs/universe";

service UniverseSkyboxService {
	rpc Render (UniverseSkyboxRenderRequest) returns (UniverseSkyboxRenderResponse) {}
}

message UniverseSkyboxRenderRequest {
	string dest_path = 1;
}

message UniverseSkyboxRenderResponse {

}